app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsViewModel.kt

```diff
@@ -0,0 +1,27 @@
+package org.oppia.app.administratorcontrols
+
+import androidx.databinding.ObservableList
+import androidx.lifecycle.ViewModel
+import org.oppia.app.administratorcontrols.administratorcontrolsitemviewmodel.AdministratorControlsApplicationSettingsViewModel
+import org.oppia.app.administratorcontrols.administratorcontrolsitemviewmodel.AdministratorControlsDownloadPermissionsViewModel
+import org.oppia.app.administratorcontrols.administratorcontrolsitemviewmodel.AdministratorControlsGeneralProfileViewModel
+import org.oppia.app.administratorcontrols.administratorcontrolsitemviewmodel.AdministratorControlsItemViewModel
+import org.oppia.app.fragment.FragmentScope
+import org.oppia.app.viewmodel.ObservableArrayList
+import javax.inject.Inject
+
+/** [ViewModel] for [AdministratorControlsFragment]. */
+@FragmentScope
+class AdministratorControlsViewModel @Inject constructor() : ViewModel() {
```

Comment:
Make this an `ObservableViewModel` instead of `ViewModel`

Reply:
Done